src/GraphQL/Execution/ExecutionStrategy.cs

```diff
@@ -50,6 +50,11 @@ protected virtual IObjectGraphType GetOperationRootType(ExecutionContext context
             {
                 case OperationType.Query:
                     type = context.Schema.Query;
+                    // note that per the GraphQL specification, a query root type is required; but currently
+                    // the schema validation check is disabled to allow for test schemas
```

Comment:
Where is this check? I don't remember but I did something with it.

> to allow for test schemas 

I think not only for this, don't remember 😞

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/blob/b6ef5f2d94dd1f22ac6cd3cad5f3bb98ee16b4e0/src/GraphQL/Types/Collections/SchemaTypes.cs#L231-L233

Reply:
Ah, yes. Exactly this place. Thanks.